autosave/mixins.py

```diff
@@ -168,7 +169,7 @@ def autosave_media(self, obj=None, get_params=''):
 
         return forms.Media(js=(
             reverse('admin:%s_%s_autosave_js' % info, args=[pk]) + get_params,
-            "%sautosave/js/autosave.js?v=2" % settings.STATIC_URL,
+            static('autosave/js/autosave.js'),
```

Comment:
Ah, I didn't know there was magic around forms media — good to know!

The `?v=2` is important to you because you aren't using the storage backend to version every static file — I am. But if it'll work anyway I'll just drop this.